src/DateRangePicker/Calendar.tsx

```diff
@@ -193,7 +194,7 @@ class Calendar extends React.Component<any, CalendarState> {
 
   renderDays(classes: any) {
     // TODO: Split this logic into smaller chunks
-    const { range, minDate, maxDate, disableDaysBeforeToday, specialDays, dateTooltip } = this.props
+    const { range, minDate, maxDate, minRangeLength, disableDaysBeforeToday, specialDays, dateTooltip } = this.props
```

Comment:
wouldn't adding default 0(`minRangeLength = 0`) to `minRangeLength` reduce a conditional below ? Meaning you wouldn't need `minRangeLength &&`.
In general I wouldn't bother much about it, but the conditional below seems a bit convoluted. So

Reply:
you mean that `minRangeLength &&` will be covered by last: `Math.abs(differenceInDays(dayMoment, range.startDate)) < minRangeLength`? 

If so, I think this is not best solution, because in this case every time all conditions will be checked, and we will always calculate `differenceInDays` which is not free of charge. In 99% of cases  `minRangeLength` will be undefined, so condition will break on first check.

Reply:
Hm, oh yea, it makes sense. 👍